modules/snooker/src/main/ts/ephox/snooker/queries/Direction.ts

```diff
@@ -0,0 +1,18 @@
+import { Fun } from '@ephox/katamari';
```

Comment:
I'm still not really sure this makes sense to be here... this I feel is more suited to Sugar.

Reply:
I considered it, but this is the only time in the monorepo that onDirection is used like this so I'm not sure it makes sense to abstract it (yet)? Everything else that calls onDirection passes something completely different for ltr and rtl.

Reply:
The problem is you're making this part of the public API of snooker. As such we need to take into account where and how it may be used in the future, rather than how it's being used directly at the moment.

Reply:
I'd prefer this stay in TinyMCE instead of making it part of Snooker if you don't want it in Sugar.

Reply:
Hm, point. I just had a better look at where this is used, and it might actually make more sense to delete this file tbh. The place it's used in the table plugin a) needs changing anyway and b) should probably just use Sugar's existing function, and it's only actually used in one file in Snooker (and is currently being passed in on init of that file everywhere _its_ called which seems silly). Unless we think it's necessary to keep this as a thing that is passed in to TableDirection? I can't see why.

Reply:
Agreed :+1: